src/runtime/Python.Runtime.csproj

```diff
@@ -19,12 +19,6 @@
     <DefineConstants>$(DefineConstants);$(ConfiguredConstants)</DefineConstants>
   </PropertyGroup>
 
-  <ItemGroup Condition=" '$(PythonInteropFile)' != '' ">
-    <Compile Remove="interop*.cs" />
-    <Compile Include="interop.cs" />
-    <Compile Include="$(PythonInteropFile)" />
```

Comment:
I am unsure removing explicit `Include` for `$(PythonInteropFile)` does not break the generation and inclusion `NativeTypeOffset`. Can you verify before we merge?

Reply:
How should I verify that?  Isn't this already checked by CI?

Reply:
In the absence of `NativeTypeOffset` the tests might still pass, but we must be sure it is actually generated on install. One option is to simply add a test, that checks presence of `NativeTypeOffset` (maybe except Windows).

Reply:
An even simpler option is to change ABI.Initialize to require NativeTypeOffset (except Windows), if that is the intended behavior.

Reply:
That is actually unclear. For the time being this just reproduces existing Python.NET behavior (e.g. 2.X) in regards to offsets. But in the upcoming release we plan to make platform-independent wheels, and generating `NativeTypeOffset` should be just an option for peculiar platforms.

Reply:
So then what are you recommending for this PR?  A test that fails when `NativeTypeOffset` is missing would have the same problem.

Reply:
I think it makes sense to have a test that fails when `NativeTypeOffset` is missing. It must be passing in CI (again I am not sure if that's true for Windows). Outside CI we could have it simply ignored. See [GitHub support](https://github.com/actions/runner/issues/368)